client/src/account_address.rs

```diff
@@ -0,0 +1,89 @@
+use std::{fs, str};
+
+use clap::{App, Arg, ArgMatches, SubCommand};
+
+use casper_types::{AsymmetricType, PublicKey};
+
+use crate::{command::ClientCommand, common};
+
+/// This struct defines the order in which the args are shown for this subcommand's help message.
+enum DisplayOrder {
+    Verbose,
+    Key,
+}
+
+/// Handles providing the arg for and retrieval of the key.
+mod key {
+    use casper_node::crypto::AsymmetricKeyExt;
+    use casper_types::AsymmetricType;
+
+    use super::*;
+
+    const ARG_NAME: &str = "public-key";
+    const ARG_SHORT: &str = "pk";
```

Comment:
I don't think this will work - did you try it?  The short args are always single chars afaik.

Either way, could you change it to `p` as per the task description please?

Reply:
Change to simply `p`